docs/user-manual/en/resource-adapter.md

```diff
@@ -0,0 +1,130 @@
+# Resource Adapter
+
+For using Apache ActiveMQ Artemis in a JavaEE/JakartaEE environment, you can use the ResourceAdapter. Before you 
+start, read carefully the [JMS chapter](using-jms.md) 
+
+##In which case do you have to use a RA and in which not?
+The main reason is the requirement of 
+[XA](https://jakarta.ee/specifications/transactions/2.0/jakarta-transactions-spec-2.0.html), distributed 
+transactions over multiple resources. If it is important for you no message get lost and multiple resources (e.g. 
+database and messaging) must be in sync, there is no way around it.
+
+##Versions
+let's give you a short overview of the versions, to be sure you pick the correct *.rar.
+
+| | artemis-ra-rar   |     |     |
+| --------- | ------ | --- | --- |
+| Artemis   | JavaEE | JCA | JMS |
+| \>= 2.18.0 | \>=8      | 1.7 | 2.0 |
+| <= 2.17.0 | 7      | 1.5 | 2.0 |
+
+| | artemis-jakarta-ra-rar | | |
+| --- | ------- | --- | --- |
+| Artemis | JavaEE | JCA | JMS |
+| \>= 2.18.0 | \>=9 | 2.0 | 3.0 |
+
+## Lets start
+To use the RA, you have to build it by your own. This sounds harder than it is. But no worries, an 
+[examples](examples.md) 
+is shipped with the distribution.
+
+* [install maven](https://maven.apache.org/install.html)
+
+```shell
+cd examples/features/sub-modules/{artemis-jakarta-ra-rar,artemis-ra-rar}
+mvn clean install
+cd target
+mv artemis*.rar artemis.rar
+```
+Now you can see the artemis.rar, and you are good to go. Follow the manual of your application server, to install the
+ResourceAdapter.
+
+## Details about the Resource Adapter configuration
+
+Before you start with the configuration you have to know two basics: The configuration is split into two 
+parts. First, the config to send messages to an address (outbound), and second, the config to get messages consumed 
+from a destination (inbound). Each can be configured separately, or use both the ResourceAdapter settings.
+
+Here are a few options listed. If you want an overview of all configuration options, consider
+[ConnectionFactoryProperties](https://github.com/apache/activemq-artemis/blob/main/artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ConnectionFactoryProperties.java)
```

Comment:
Linking to main kind of sucks, and may not apply to whatever version is actually being looked at. Will whats wanted it be in the javadoc?

Reply:
of all your comments, this hurts the most :D In my opinion, link the src, is the best you could do, because the docu can't be outdated. With two mouseclicks you can select the tag version of the readers version and you are fine. Is link the src a no go for you? Or is it ok for you to add a hint, to select the right version of the src file?

Reply:
The docs are meant to be version specific, so linking to main for something that is documenting not-main, indeed kind of sucks. Sure, you can then change the version, but you can also fail to do so, which history suggests to me happens a whole bunch. We also aren't meant to direct regular everyday users to unreleased software as a matter of policy, which this somewhat skirts a little.

I asked about the javadoc, as it could then at least link to doc for the latest _released_ version, if not somehow the actual specific version (e.g I'd wonder if there wasnt some way to filter the version in), though I guess the latter would equally apply to the src.